libindy/src/services/wallet/default.rs

```diff
@@ -30,10 +30,18 @@ impl Default for DefaultWalletRuntimeConfig {
 }
 
 #[derive(Deserialize)]
-struct DefaultWalletCredentials {}
+struct DefaultWalletCredentials {
+    key: String
+}
 
 impl<'a> JsonDecodable<'a> for DefaultWalletCredentials {}
 
+impl Default for DefaultWalletCredentials {
```

Comment:
We need to update documentation for indy_create_wallet/indy_open_wallet functions and describe format of credentials param. We need it in:

- api/wallet.rs comments
- include/wallet.h comments
- python wrapper
- java wrapper
- iOS wrapper
- .Net wrapper too.